### ISSUE TITLE
Code view LoC numbers don't match the ramp view LoC numbers

### ISSUE THREAD

**Issue by damithc** [reposense/RepoSense#1843]
Example https://nus-cs2113-ay2223s1.github.io/tp-dashboard

![image](https://user-images.githubusercontent.com/1673303/199912234-0442bdb4-27d8-45b4-a813-ab3f46a72fa0.png)

**Comment by Zhou-Jiahao-1998** [comment]
@damithc Hi Prof. Sorry for the delay. I traced back the code and notice that this implementation has been here since Aug 2021 from https://github.com/reposense/RepoSense/commit/c374fc45840a63222b36a5fe993e54ddf0684929. I am not sure if this was done on purpose.

Using the above screenshot as an example, the labels for selection on top counts **ALL** lines in the files that the author has written to. However, the labels on each files correctly display the LoC written by the author. The LoC number displayed above the ramp is the sum of all the LoC of each file which is also correct.

I checked the [documentation](https://reposense.org/RepoSense/dg/report.html#html-report) and the screenshot given has matching LoC on both sides. Therefore, I think it is more likely that it is a bug rather than a feature.

Do let me know if this is urgent. If it is urgent, I can push out a fix before school starts officially so that it does not affect the modules using it next semester. Otherwise, I think it can be a task for the juniors to work on.
